examples/getting-started-session-based/03-serving-session-based-model-torch-backend.ipynb

```diff
@@ -70,8 +70,56 @@
      "text": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for col_name, col_schema in input_schema.column_schemas.items():
</pre><p>so this cell is needed to tell Ensemble api that model does not consume ragged inputs since value count min and max are same?  would <code>input_schema</code> wont print out required properties without executing this cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/631/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Yeah, that's right. This is a manual step to capture the info from the `sparse_max` dictionary in the schemas, which helps the serving code know the shapes of the inputs we ended up with after padding in the Workflow.

Reply:
In the longer term, we'll probably want to re-evaluate this and see if we can migrate from `sparse_max` to capturing shapes in the model schemas (which you're already working on the first part of.)